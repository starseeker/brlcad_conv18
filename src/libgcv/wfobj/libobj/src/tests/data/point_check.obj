v 0 0 0
v 0 1 0
v 1 1 0
v 1 0 0

vt 10 10 10
vt 10 11 10
vt 11 11 10
vt 11 10 10

vn 20 20 20
vn 20 21 20
vn 21 21 20
vn 21 20 20

p 1
p 1 3

v 0 0 1
v 0 1 1
v 1 1 1
v 1 0 1

vt 10 10 11
vt 10 11 11
vt 11 11 11
vt 11 10 11

vn 20 20 21
vn 20 21 21
vn 21 21 21
vn 21 20 21

p 5 6 7
p 1 2 3 4 5 6 7
