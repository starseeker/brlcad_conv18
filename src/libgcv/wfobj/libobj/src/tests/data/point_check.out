vertices {
  <0,0,0,1>
  <0,1,0,1>
  <1,1,0,1>
  <1,0,0,1>
  <0,0,1,1>
  <0,1,1,1>
  <1,1,1,1>
  <1,0,1,1>
}
texture coordinates {
  <10,10,10>
  <10,11,10>
  <11,11,10>
  <11,10,10>
  <10,10,11>
  <10,11,11>
  <11,11,11>
  <11,10,11>
}
normals {
  <20,20,20>
  <20,21,20>
  <21,21,20>
  <21,20,20>
  <20,20,21>
  <20,21,21>
  <21,21,21>
  <21,20,21>
}
point {
  groups: default
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1 
}
point {
  groups: default
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1 3 
}
point {
  groups: default
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5 6 7 
}
point {
  groups: default
  object: ''
  material: ''
  materiallibs: 
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1 2 3 4 5 6 7 
}
